src/v3/src/OktaSignIn/index.ts

```diff
@@ -159,14 +172,15 @@ export default class OktaSignIn {
         if (target) {
           // @ts-ignore OKTA-508744
           render(h(Widget, {
+            ...this.options,
             authClient: this.authClient,
             globalSuccessFn: onSuccessWrapper,
             globalErrorFn: onErrorWrapper,
             eventEmitter: this.eventEmitter,
-            ...this.options,
+            widgetHooks: this.widgetHooks,
```

Comment:
Putting `eventEmitter` and `widgetHooks` after `...this.options` is needed to prevent lint error `This spread always overwrites this property.`
I looks fine to put spread at the top as we don't expect overwriting any of listed properties (`authClient`, `globalSuccessFn`, `globalErrorFn `, `eventEmitter` and `widgetHooks`) from options.

Reply:
Ok, yeah I'm convinced now that in fact we shouldn't spread at the bottom of the object anyways since we don't want to overwrite the `authClient` (or any of the other ones, actually) accidentally since it's always set in the constructor function.

Reply:
Actually we're kind of spreading in extra properties to the `Widget` component we don't need, like `onSuccess`, since it's still in the `options` object after we add the wrappers around them. Not a big deal and no change needed for your PR but realizing we could clean this up a bit.